unit-tests/live/d400/test-auto-limits.cpp

```diff
@@ -6,8 +6,6 @@
 /////////////////////////////////////////////////////////////////////////////
 
 //#test:device D400* !D457
-// auto-limits option is deprecated currently [LRS-487]
-//#test:donotrun
 
 #include "../../catch.h"
```

Comment:
Won't it be better to change to a python test?

Reply:
Done